site/e2e/helpers.ts

```diff
@@ -885,3 +885,17 @@ export async function openTerminalWindow(
 
   return terminal;
 }
+
+/**
+ * Ensures that no uncaught errors creep through the UI.
+ *
+ * Likely most useful for asserting that nothing breaks when switching from the
+ * JS-based runtime for the test environments to the Go-based runtime for the
+ * production backend.
+ */
+export async function assertNoUncaughtRenderError(page: Page): Promise<void> {
```

Comment:
nit: maybe this function could have a more friendly name like `assertNoUnexpectedUIError`. I think the word `Uncaught` is not very common at least for me.